controller/cmd/service-mirror/cluster_watcher.go

```diff
@@ -297,14 +297,14 @@ func (rcsw *RemoteClusterServiceWatcher) cleanupMirroredResources() error {
 		return RetryableError{[]error{innerErr}}
 	}
 
-	for _, endpt := range endpoints {
-		if err := rcsw.localAPIClient.Client.CoreV1().Endpoints(endpt.Namespace).Delete(endpt.Name, &metav1.DeleteOptions{}); err != nil {
+	for _, endpoint := range endpoints {
+		if err := rcsw.localAPIClient.Client.CoreV1().Endpoints(endpoint.Namespace).Delete(endpoint.Name, &metav1.DeleteOptions{}); err != nil {
```

Comment:
I see no particular reason for this abbreviation (and the longer form is used elsewhere)